psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -72,37 +75,37 @@ PSDKWrapper::on_configure(const rclcpp_lifecycle::State &state)
   {
     return CallbackReturn::FAILURE;
   }
-  initialize_ros_elements();
-
-  return CallbackReturn::SUCCESS;
-}
-
-PSDKWrapper::CallbackReturn
-PSDKWrapper::on_activate(const rclcpp_lifecycle::State &state)
-{
-  (void)state;
-  RCLCPP_INFO(get_logger(), "Activating PSDKWrapper");
-
   T_DjiUserInfo user_info;
   set_user_info(&user_info);
 
   if (!init(&user_info))
   {
     return CallbackReturn::FAILURE;
   }
-
-  activate_ros_elements();
-
   if (!init_telemetry() || !init_flight_control() || !init_camera_manager() ||
       !init_gimbal_manager() || !init_liveview())
```

Comment:
Here I would suggest to filter which ones are essential for flying/operate: 
- `telemetry`
- `flight_control` 
-  `liveview`? (I think all the drones have liveview for the FPV camera)  
 
Leave the rest as optional. Simply notify as some optional modules were not loaded or initialized. I believe it tackles a concern from our users too

Reply:
done here https://github.com/umdlife/psdk_ros2/pull/34/commits/e67485f505afac84a6d6dc3e26d26b221c1f9e5a